lib/client.js

```diff
@@ -61,7 +67,24 @@ class Client {
 		}
 		args.path = `${PATH_PREFIX}/channels/videos/list`;
 		args.query = {
-			channel_key: id // eslint-disable-line camelcase
+			channel_key: id
+		};
+		return this.makeRequest(args);
+	}
+
+	// args.playlistId *required
+	v2getVideosByPlaylist(args) {
```

Comment:
Prolly better to just rip it out... this is what the version bump is for.

Plus I like deleting things. :-)